references/cli/addon-registry.go

```diff
@@ -79,6 +79,22 @@ add a gitlab registry: vela addon registry add my-repo --type gitlab --endpoint=
 			if err != nil {
 				return err
 			}
+			if registry.Helm == nil {
+				_, err = registry.ListAddonMeta()
```

Comment:
CI fails, because using this function to check can cause many github api requests that exceed the API limit.

Do we have any more efficient way?